spec/project-find-view-spec.js

```diff
@@ -1369,6 +1369,11 @@ describe('ProjectFindView', () => {
 
       describe("when the find field contains a ^ or a $ and the regex option is enabled", () => {
         it("correctly replaces all matches", async () => {
+          // TODO: Remove version check when Atom 1.21 reaches stable
+          if (parseFloat(atom.getVersion()) < 1.21) {
```

Comment:
I'm aware.  Atom 1.9 was released approximately one year ago and is not supported.